config/puma.rb

```diff
@@ -0,0 +1,14 @@
+preload_app!
```

Comment:
I would use default config from Rails 5 and tune if needed (good to remove hard-coded values).

P.S. Registry project also needs this to be fixed.

https://gist.github.com/artur-beljajev/c7009c913e52f83b07a5d9d7ac5eecc1